src/components/visual/explorer-link/explorer-link.tsx

```diff
@@ -3,9 +3,11 @@ import { Component, Element, h, Prop, State } from '@stencil/core';
 import { getIconHtmlFromIconDefinition } from 'utils/icons/getIconHtmlFromIconDefinition';
 import { getIconHtmlFromIconName } from 'utils/icons/getIconHtmlFromIconName';
 
+import { EXPLORER_LINK_SLOT_NAME } from './explorer-link.constants';
+
 @Component({
   tag: 'mvx-explorer-link',
-  styleUrl: 'explorer-link.css',
+  styleUrl: 'explorer-link.scss',
```

Comment:
why not shadow true ?

Reply:
Will be solved in future `Pull Request`. 👍